src/utils/resources/template/utils/selectors.ts

```diff
@@ -52,6 +52,22 @@ export const getTemplateProviderName = (template: V1Template): string =>
 export const getTemplateSupportLevel = (template: V1Template): string =>
   getAnnotation(template, ANNOTATIONS.supportLevel);
 
+/**
+ * A selector that returns the containerDisks of a given template
+ * @param template - template
+ * @returns array of containerDisks
+ */
+export const getTemplateContainerDisks = (template: V1Template): string | undefined =>
+  getAnnotation(template, ANNOTATIONS.containerDisks);
+
+/**
+ * A selector that returns the import URLs of a given template
+ * @param {V1Template} template - template
+ * @returns array of import URLS
+ */
+export const getTemplateImportURLs = (template: V1Template): string | undefined =>
+  getAnnotation(template, ANNOTATIONS.importURLs);
+
```

Comment:
yes, it's a list encoded as a string with EOL seperators

```suggestion
 * @returns {[]string} array of import URLS
 */
export const getTemplateImportURLs = (template: V1Template): []string =>
  template.metadata?.annotations['template.kubevirt.io/images']?.split('\n') || [];

```

Reply:
Okay thanks :-)